kuksa_databroker/databroker/src/main.rs

```diff
@@ -183,16 +236,21 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 .value_name("PORT")
                 .required(false)
                 .env("KUKSA_DATA_BROKER_PORT")
+                .value_parser(clap::value_parser!(u16).range(1..))
                 .default_value("55555"),
         )
         .arg(
             Arg::new("metadata")
                 .display_order(4)
                 .long("metadata")
-                .help("Populate data broker with metadata from file")
+                .help("Populate data broker with metadata from (comma-separated) list of files")
                 .takes_value(true)
+                .use_value_delimiter(true)
+                .require_value_delimiter(true)
+                .multiple_values(true)
```

Comment:
I think it would be beneficial to be able to specify `--metadata FILE` multiple times (in addition to a comma-separated list of files).
```suggestion
                .multiple_values(true)
                .action(clap::ArgAction::Append)
```